vision/cloud-client/detect/beta_snippets.py

```diff
@@ -176,8 +177,8 @@ def detect_handwritten_ocr_uri(uri):
 # [END vision_handwritten_ocr_gcs_beta]
 
 
-# [START vision_fulltext_detection_pdf_beta]
-def detect_document_features(path):
+# [START detect_batch_annotate_files_beta]
```

Comment:
Keep the `vision_` at the start of the region tags.

Reply:
And for the ones below.

Reply:
Not sure what you mean. Can you elaborate a bit more? You mean something like beta_detect_batch_annotate_files?

Reply:
```
# [vision_detect_batch_annotate_files_beta]
```
We keep the product name at the start of every region tag, so that sample tracker can pick it up.

Reply:
oops. I read it VERSION the first time. Good catch.  Fixed...